content/03.text-mining.md

```diff
@@ -148,7 +148,7 @@ Future clustering approaches should consider simplifying sentences to prevent th
 Overall unsupervised methods provide a means to rapidly find relationship asserting sentences without the need of annotated text.
 Approaches in this category range from using co-occurrence scores to clustering sentences.
 These methods provide a generalizable framework that can be used on large repositories of text.
-Future methods can improve detection power by considering the use of  methods that simplify sentences and use datasets that include full text articles.
+Future methods can improve detection power by considering the use of methods that simplify sentences and use datasets that include full text articles.
```

Comment:
```suggestion
Future methods can improve detection power by considering the use of methods that simplify sentences and use datasets that include full text articles.
```
Do you want to cite the paper that talks about cooccurrences improving with full text? Something like:
> Full text has already been show to make a dramatic improvements for methods that aim to infer relationships using cooccurrences[ref], and we should expect similar benefirts for machine learning approaches.